src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -148,7 +148,7 @@ public function all($namespace = null)
      */
     public function getLongVersion()
     {
-        return parent::getLongVersion().sprintf(' (env: <comment>%s</>, debug: <comment>%s</>)', $this->kernel->getEnvironment(), $this->kernel->isDebug() ? 'true' : 'false');
+        return parent::getLongVersion().sprintf(' (env: <comment>%s</>, debug: <comment>%s</>) <bg=#0057B7;fg=#FFDD00>#StandWith</><bg=#FFDD00;fg=#0057B7>Ukraine</> <href=https://sf.to/ukraine>Support Ukraine</>', $this->kernel->getEnvironment(), $this->kernel->isDebug() ? 'true' : 'false');
```

Comment:
```suggestion
        return parent::getLongVersion().sprintf(' (env: <comment>%s</>, debug: <comment>%s</>) <bg=#0057B7;fg=#FFDD00;href=https://sf.to/ukraine>#StandWith</><bg=#FFDD00;fg=#0057B7;href=https://sf.to/ukraine>Ukraine</>', $this->kernel->getEnvironment(), $this->kernel->isDebug() ? 'true' : 'false');
```

Reply:
I'm not sure about merging the banner and the link. Terminals display links in a very subtle way, so maybe it's not as visible as showing the banner and the link separately.